Source/WebCore/platform/audio/cocoa/CARingBuffer.cpp

```diff
@@ -161,14 +161,17 @@ static void StoreABL(Vector<Byte*>& pointers, size_t destOffset, const AudioBuff
 static void FetchABL(AudioBufferList* list, size_t destOffset, Vector<Byte*>& pointers, size_t srcOffset, size_t nbytes, AudioStreamDescription::PCMFormat format, CARingBuffer::FetchMode mode)
 {
     ASSERT(list->mNumberBuffers == pointers.size());
```

Comment:
Remove the assert?

Reply:
I think it's still an exceptional state that should be caught by an ASSERT.  The intent of this patch is to make this state fail gracefully in release builds, however.